tests/test_portchannel_global.rb

```diff
@@ -146,6 +177,8 @@ def test_load_balance_no_rotate
     assert_equal(
       global.default_symmetry,
       global.symmetry)
+  rescue Cisco::CliError => e
+    check_and_raise_error(e, 'This command is only supported on T2 platforms.')
```

Comment:
I don't think raising an error here is appropriate. You really only want this test to run when:

1) It's a T2 (i.e. 31xx)
2) It's running in 3k switch-mode

Which means you need a T2 helper method and then:

```
     skip('Test not supported on this platform') unless n3k_is_t2 && n3k_in_n3k_mode?
```

For the t2 helper you can either create a method in ciscotest to scape it from show ver, or add a new property in `show_version.yaml`:

```
chassis_id:
  get_value: "chassis_id"       # Nexus 3172 Chassis
```

I prefer the second option since I think we'll need this for other properties eventually, and it could be used by beaker as well. But right now it's just affecting some tests so a simple scraper method is fine too.